compiler/rustc_target/src/spec/solid_base.rs

```diff
@@ -5,6 +5,7 @@ pub fn opts(kernel: &str) -> TargetOptions {
     TargetOptions {
         os: format!("solid_{}", kernel).into(),
         vendor: "kmc".into(),
+        executables: false,
```

Comment:
cc @kawadakk
Is this intentional?
The target specifies a linker, but doesn't specify support for executables or dynamic libraries, so the linker is never used, which is weird.

Reply:
Yes, it's intentional. This target requires an external toolchain (linker + post-processor + runtime libraries) to build functional executables or dynamic libraries.

Reply:
Wait, what?

That's not the impression I had when that target was first approved?

What is the license on that toolchain?

Reply:
>This target requires an external toolchain (linker + post-processor + runtime libraries) to build functional executables or dynamic libraries.

That's true for majority of our targets though, linker/libc/startup objects are typically not shipped with Rust toolchain.
`rustc` just calls the linker and passes libraries to it by name and expects them to be there.
Although I'm not sure what the post-processor step means for this target.

Reply:
Ok, keeping `executables: false` for these targets.